backend/tests/data/test.txt

```diff
@@ -0,0 +1,37 @@
+# test taxonomy
```

Comment:
FYI I have an updated version of test.txt in openfoodfacts-server, with some more complex DAG. https://github.com/openfoodfacts/openfoodfacts-server/blob/main/taxonomies/test.txt